### ISSUE TITLE
Integrate turn-by-turn navigation module

### ISSUE THREAD

**Issue by fabianbormann** [nav-e/nav-e#3]
During GSoC our student Thomas implemented a turn-by-turn navigation (https://github.com/Greennav/gn-path-follower) :+1:  but it is not integrated yet.
Maybe the gn-turn-by-turn (see modules) navigation could be accessible via a paper-button - or something like this - in the gn-menu.

**Comment by fabianbormann** [comment]
falsely closed by old commit messages from other repos :sweat_smile:

**Comment by sumeetkhirwal** [comment]
@fabianbormann  Is this project active ??

**Comment by fabianbormann** [comment]
@sumeetkhirwal yes it's active :wink: so if you want to implement the turn-by-turn navigation -> do it ! :+1:

**Comment by shreyakupadhyay** [comment]
- @fabianbormann Does [routing](https://github.com/nav-e/routing) (go backend server) supports turn-by-turn navigation and provides this information? 
- If go backend server does not provide this feature OpenLayers Google-Map-Integration can be used for getting turn-by-turn details as with new version it provides support with google maps. I will start working on this, once updated. Thanks.

**Comment by maxlorenz** [comment]
@shreyakupadhyay it doesn't have that yet, but we can access the [information](https://wiki.openstreetmap.org/wiki/Key:turn).  I opened an issue there :)
